go-controller/pkg/util/util.go

```diff
@@ -137,6 +138,111 @@ func HashForOVN(s string) string {
 	return fmt.Sprintf("a%s", hashString)
 }
 
+type NetNameInfo struct {
+	// netconf's name
+	NetName     string
+	Prefix      string
+	IsSecondary bool
+}
+
+type NetAttachDefInfo struct {
+	NetNameInfo
+	// net-attach-defs shared the same CNI Conf, key is <Namespace>/<Name> of net-attach-def.
+	// Note that it means they share the same logical switch (subnet cidr/MTU etc), but they might
+	// have different resource requirement (requires or not require VF, or different VF resource set)
+	NetAttachDefs sync.Map
+	NetCidr       string
+	MTU           int
+	TopoType      string
+	VlanId        int
+	ExcludeIPs    []net.IP
+}
```

Comment:
I think these structs should be defined in a dedicated package, instead of a utils file with a bunch of unreleated code.